scripts/spack/packages/serac/package.py

```diff
@@ -298,6 +341,15 @@ def cache_name(self):
             special_case,
         )
 
+    def initconfig_compiler_entries(self):
+        spec = self.spec
+        entries = super().initconfig_compiler_entries()
+
+        # Add optimization flag workaround for Debug builds with cray compiler or newer HIP
+        if "+rocm" in spec:
+            entries.append(cmake_cache_string("CMAKE_CXX_FLAGS_DEBUG", "-O1 -g -DNDEBUG"))
```

Comment:
This is bizaare.. not saying its wrong but `-O1` on a debug build is weird.

Reply:
i should investigate whether this is actually needed. this is another thing i just grabbed from axom's spack package.

Reply:
i ran the tests while on debug and commenting out this line in the host config, and it appeared to pass everything. ill just remove it.